java/src/main/java/com/logicalclocks/hsfs/engine/FeatureViewEngine.java

```diff
@@ -28,135 +45,53 @@
 
 import java.io.IOException;
 import java.text.ParseException;
+import java.util.ArrayList;
 import java.util.Date;
 import java.util.List;
 import java.util.Map;
 import java.util.stream.Collectors;
 
-import static com.logicalclocks.hsfs.TrainingDatasetType.IN_MEMORY_TRAINING_DATASET;
+public class FeatureViewEngine extends FeatureViewEngineBase {
 
-public class FeatureViewEngine {
+  private static final Logger LOGGER = LoggerFactory.getLogger(FeatureViewEngine.class);
```

Comment:
Best if we move it to the base class (check previous comments for an explanation).